src/game_renderer/GameRenderer.cpp

```diff
@@ -83,7 +79,7 @@ void GameRenderer::Init(const std::string& resource_dir,
   std::cout << "OpenGL version: " << glGetString(GL_VERSION) << std::endl;
   std::cout << "GLSL version: " << glGetString(GL_SHADING_LANGUAGE_VERSION)
             << std::endl;
-  glfwSwapInterval(0);  // vsync
+  glfwSwapInterval(1);  // vsync
```

Comment:
Would you guys like it if I disabled vsync if DEBUG is set for performance testing?